src/senaite/jsonapi/configure.zcml

```diff
@@ -126,16 +126,19 @@
       />
 
   <!-- Adapter for Bika LIMS Proxy Field -->
-  <adapter
-      for="bika.lims.interfaces.IProxyField"
-      factory=".fieldmanagers.ProxyFieldManager"
-      />
-
-  <!-- Adapter for Bika LIMS AR Analyses Field -->
-  <adapter
-      for="bika.lims.interfaces.IARAnalysesField"
-      factory=".fieldmanagers.ARAnalysesFieldManager"
-      />
+  <configure zcml:condition="installed senaite.core">
+      <adapter
+          for="bika.lims.interfaces.IProxyField"
+          factory=".fieldmanagers.ProxyFieldManager"
```

Comment:
This field manager is also available in Bika LIMS, so you can leave it available per default.